.github/workflows/vast.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
           CC: gcc-8
           CXX: g++-8
         run: |
-          ./configure --with-bundled-caf --without-arrow
+          cmake -B build -DVAST_ENABLE_BUNDLED_CAF=1 -DVAST_ENABLE_ARROW=0
```

Comment:
Can you use -DVAR:TYPE=VALUE so the proper type shows up in `ccmake`?

E.g.,

`-DVAST_ENABLE_BUNDLED_CAF:BOOL=ON`

Reply:
Sure, although this is the CI config so there is no human to run `ccmake` later on ;)